tests/cli_test/delete_test.py

```diff
@@ -0,0 +1,78 @@
+import unittest
+from argparse import Namespace
+from unittest.mock import MagicMock, patch
+
+from serverless_llm.cli.delete import DeleteCommand
+
+
+class TestDeleteCommand(unittest.TestCase):
+    @patch("serverless_llm.cli.delete.requests.post")
+    def test_delete_single_model_success(self, mock_post):
+        # Mock the response of the requests.post
+        mock_response = MagicMock()
+        mock_response.status_code = 200
+        mock_post.return_value = mock_response
+
+        args = Namespace(models=["facebook/opt-1.3b"])
+        command = DeleteCommand(args)
+        command.run()
+
+        mock_post.assert_called_once_with(
+            "http://localhost:8343/delete/",
+            headers={"Content-Type": "application/json"},
+            json={"model": "facebook/opt-1.3b"},
+        )
+        self.assertEqual(mock_post.return_value.status_code, 200)
```

Comment:
As we could not have a mock server that receives POST request, so I just want to pretend there was a successful response. So the assert of status code is just a regular assert, if there is error during the command running, the tests would not passed, though it may not be the assert error. Do you think I can just remove the assert of the status code?